README.rst

```diff
@@ -124,9 +124,10 @@ Software using PyMC3
 --------------------
 
  - `Bambi <https://github.com/bambinos/bambi>`__: BAyesian Model-Building Interface (BAMBI) in Python.
- - `NiPyMC <https://github.com/PsychoinformaticsLab/nipymc>`__: Bayesian mixed-effects modeling of fMRI data in Python
- - `gelato <https://github.com/ferrine/gelato>`__: Bayesian Neural Networks with PyMC3 and Lasagne
- - `beat <https://github.com/hvasbath/beat>`__: Bayesian Earthquake Analysis Tool
+ - `NiPyMC <https://github.com/PsychoinformaticsLab/nipymc>`__: Bayesian mixed-effects modeling of fMRI data in Python.
+ - `gelato <https://github.com/ferrine/gelato>`__: Bayesian Neural Networks with PyMC3 and Lasagne.
+ - `beat <https://github.com/hvasbath/beat>`__: Bayesian Earthquake Analysis Tool.
+ - `Edward <https://github.com/blei-lab/edward>`__: A library for probabilistic modeling, inference, and criticism.
```

Comment:
I suppose they're still there, just deprecated. I'm OK with this then if you want to add it.